be/src/exec/arrow_to_starrocks_converter.cpp

```diff
@@ -727,6 +729,87 @@ struct ArrowConverter<AT, LT, is_nullable, is_strict, JsonGuard<LT>> {
     }
 };
 
+template <ArrowTypeId AT, LogicalType LT, bool is_nullable, bool is_strict>
+struct ArrowConverter<AT, LT, is_nullable, is_strict, VariantGuard<LT>> {
```

Comment:
This is used for load parquet variant to SR varinat column? I think you can add these later, this PR we can foucus on column defintion